File: saveInf.go
package main

import (
	"encoding/json"
	"fmt"
	"io/ioutil"
	"math"
)

type _Data struct {
	Volume float64
}

func main() {
	var data _Data
	data.Volume = math.Inf(1)
	substr, err := json.Marshal(data)
	if err != nil {
		fmt.Println(err)
	}
	fmt.Println(data)
	err = ioutil.WriteFile("myjsonfile.json", []byte(substr), 0644)
	if err != nil {
		fmt.Println(err)
	}

}

File: saveInf.js
var fs = require('fs');

var maxNumber = Math.pow(10, 1000);
if (maxNumber === Infinity) {
  console.log("Infinity!");
}

var obj = { volume: 0.0 }
obj.volume = maxNumber;

var json = JSON.stringify(obj);
fs.writeFileSync('myjsonfile.json', json);